image_get/README.md

```diff
@@ -0,0 +1,32 @@
+## What is this?
```

Comment:
I think it would make sense for this project to have its own repo; it's not part of the API.

(I know we have other projects in this repo too, I think it makes sense for them to have their own repos as well, but we can discuss that later).

Reply:
I agree that this doesn't fit into the API repo as well as the other projects. Let's "graduate" this to its own repository after the implementation is finished? I don't want to break my momentum with hours of documentation, container management, and CI setup.

As for the other projects, what would we gain from splitting them into individual repositories? I know it would cost many hours in the form of updating documentation, rewriting the testing pipeline, setting security credentials for each repository, and cognitive overhead.

Reply:
Let's discuss the other projects on a case by case basis when it makes sense, I don't think there's any reason to split them up any time soon. I think it probably makes sense to discuss them when we're doing significant work on them anyway.

I'm fine with graduating this project once the implementation is done.